rapier/rules-update-2020-07.md

```diff
@@ -0,0 +1,63 @@
+ 
+**Revision**: {{ site.git.last_commit.short_sha }}
+
+# Rubber-tipped Spears
+
+Atlantia now allows for the use of rubber-tipped spears in both rapier and C&T combat.  Rapier Spears are specifically precluded from use in light rapier.  For the foreseeable future, all Rapier Spear activities must be actively marshaled by a rapier marshal with a Rapier Spear authorization.
+
+All those marshaling Rapier Spear activity should record and submit detailed feedback in the Event Report as to the efficacy, safety, and any issues raised during Rapier Spear activities.
+
+The marshal-in-charge of an activity may limit Rapier Spears on a per-activity basis as long as the limitation is widely announced.  As an example, after bringing all participants together, announce: "For the next melee, you may not use Rapier Spears."
+
+# Parry-Only Daggers
+
+All equipment with a steel or rubber blade must pass the blade requirements.  Equipment without a blade is not an offensive weapon in the context of Rapier in Atlantia.  As an example, an armored combat rattan dagger does not have a blade, and therefore can be used as a parrying device, similar to the already approved wooden dowel rod or rolling pin.
+
+# Knee-Walking
+
+Knee-Walking is allowed by society policy at all times.  The marshal-in-charge of an activity may limit knee-walking on a per-activity basis as long as the limitation is widely announced.  As an example, after bringing all participants together, announce: "For the next melee, you may only knee walk to engagement."
+
+# C&T Melee
+
+The new society rules allow for melees using C&T combat.  All participants in C&T melee must hold an active C&T authorization and must be actively marshaled by someone approved to marshal C&T combat in Atlantia.  Participants with a Rapier Spear authorization without a C&T authorization may participate in C&T melees using only a Rapier Spear.
+
+All those marshaling C&T melee should record and submit detailed feedback in the Event Report as to the efficacy, safety, and any issues raised during C&T melee activities.
+
+# C&T blows to the legs
+
+In C&T, blows to the leg are no longer considered "fighting-ending".  Participants that receive a blow to the leg must follow standard rapier guidelines for blows to the leg.  As with rapier, participants that do not wish to continue after receiving a blow to the leg may choose to take any blow as a "killing" blow.
+
+# Updated guidance for armor coverage
+
+The [new society handbook](https://www.sca.org/wp-content/uploads/2020/04/FencingMarshalsHandbookApril2020.pdf) includes images that explicitly defines the minimum armor coverage required for torso armor, gorget, and mask coverage on pages 19, 20, and 21 respectively.  All participants must meet these standards.
+
+# Thrown Cloaks
+
+Cloaks may be thrown towards an opponent, but may specifically not throw cloaks over an opponent's face or with the intent to trip an opponent.  

```

Comment:
This is going to be either ugly or stupid in practice, as the only useful things to do with a cloak projectile is to obscure vision or entangle their sword.

https://3.bp.blogspot.com/-BNLahB2f6dI/VdpiSPKKqCI/AAAAAAAAGjg/mSz8NzK2EYQ/s1600/fabris%2Brapier%2Band%2Bcape2.jpg

Reply:
How is spamming the cloak to be handled? If I throw it, do I get a free hold? How many holds?

Reply:
addressed